routes/user.js

```diff
@@ -454,7 +454,7 @@ router.post("/settings/update", async function (req, res) {
             const yiq = (rgb[0] * 2126 + rgb[1] * 7152 + rgb[2] * 722) / 10000;
             const isLight = yiq >= contrastTolerance * 256;
 
-            if ((prop == "textColor" || prop == "nameColor") && isLight) {

```

Comment:
i don't know what other fuctions use this, i'd rather add prop == backgroundColour (or its equivalent) here